README.md

```diff
@@ -1,6 +1,6 @@
 # Getting Started with the Blazor Speed Dial Component
 
-A quick start Blazor project that shows how to add a Blazor Speed Dial component to the Blazor WebAssembly App. This project also includes a code snippet to set an add icon to the speed dial button, as well as how to customize it using predefined CSS classes and position options.
+The [Blazor Speed Dial](https://www.syncfusion.com/blazor-components/blazor-speeddialutm_source=github&utm_medium=listing&utm_campaign=blazor-speed-dial-github-samples) is an extension of the Blazor Floating Action Button that displays a list of action buttons when clicked.A quick start Blazor project that shows how to add a Blazor Speed Dial component to the Blazor WebAssembly App. This project also includes a code snippet to set an add icon to the speed dial button, as well as how to customize it using predefined CSS classes and position options.
```

Comment:
Recheck on all the utm parameters links

Reply:
rechecked on the utm parameters and also gave a space after the dot